docs/core/whats-new/dotnet-8/overview.md

```diff
@@ -0,0 +1,103 @@
+---
+title: What's new in .NET 8
+description: Learn about the new .NET features introduced in .NET 8.
+titleSuffix: ""
+ms.date: 11/14/2023
+ms.topic: overview
+ms.author: gewarren
+author: gewarren
+---
+# What's new in .NET 8
+
+.NET 8 is the successor to [.NET 7](../dotnet-7.md). It will be [supported for three years](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) as a long-term support (LTS) release. You can [download .NET 8 here](https://dotnet.microsoft.com/download/dotnet).
+
+## .NET runtime
+
+The .NET 8 runtime includes improvements to performance, garbage collection, and the core and extension libraries. It also includes a new globalization mode for mobile apps and new source generators for COM interop and configuration binding. For more information, see [What's new in the .NET 8 runtime](runtime.md).
+
+## .NET SDK
+
+For information about what's new in the .NET SDK, containers, Native AOT, code analysis, and diagnostics, see [What's new in the SDK and containers for .NET 8](sdk-containers.md).
+
```

Comment:
I'd like to see a link to what's new in C# as well. I think it fits here after the SDK. What do you think?

```suggestion

## C# 12

C# 12 shipped with the .NET 8 SDK. For more information, see [what's new in C# 12](../../../csharp/whats-new/csharp-12.md).

```